sdk/communication/arm-communication/README.md

```diff
@@ -6,8 +6,8 @@ REST API for Azure Communication Services
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication/arm-communication) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/arm-communication) |
-[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-communication) |
 [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-communication) |
```

Comment:
Can you move the pipe at the end to the end of Samples it's not part of the link it's just a demarcation .